nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/trash/TrashService.java

```diff
@@ -170,6 +170,14 @@ default void untrashDocument(DocumentModel doc) {
      */
     DocumentModelList getDocuments(DocumentModel parent);
 
+    /**
+     * Checks if the name of a document is mangled.
+     *
+     * @param docName the document's name
+     * @since 11.1
+     */
```

Comment:
i think you can replace the javadoc of this method by 

`@return true if the document related to the docName is mangled, otherwise false`